settings/EventsExtras.setting.php

```diff
@@ -14,8 +14,11 @@
    'is_domain' => 1,
    'is_contact' => 0,
    'description' => ts('Show/Hide Default Role'),
-   'help_text' => 'help ka',
-   'section' => 'front_page',
+   'help_text' => '',
+   'extra_attributes' => [
+    'section' => 'event_info',
```

Comment:
Can we use the constants you created here? i.e `CRM_EventsExtras_SettingsManager::EVENT_INFO`
Same for other places where you added section too.